src/main/java/io/codurance/pouch/HealthController.java

```diff
@@ -0,0 +1,32 @@
+package io.codurance.pouch;
+
+import com.fasterxml.jackson.annotation.JsonAutoDetect;
+import com.fasterxml.jackson.annotation.PropertyAccessor;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.Optional;
+
+@RestController
+public class HealthController {
+
+    @Autowired
+    HealthRepository healthRepository;
+
+    @RequestMapping(produces = MediaType.APPLICATION_JSON_VALUE,
+            method = RequestMethod.GET,
+            value = "/healthcheck")
+    public String checkHealth() throws JsonProcessingException {
+        ObjectMapper objectMapper = new ObjectMapper();
+        objectMapper.setVisibility(PropertyAccessor.FIELD, JsonAutoDetect.Visibility.ANY);
+
+        Optional<Health> healthData = healthRepository.findById(1);
+
+        return objectMapper.writeValueAsString(healthData.get());
```

Comment:
You shouldn't need to use Jackson's object mapper directly at all. We can pair up on this so I can show you some cool tricks here.